python/lsst/ts/ataos/config_schema.py

```diff
@@ -0,0 +1,227 @@
+# This file is part of ts_scheduler.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_ATAOS/blob/master/schema/ATAOS.yaml
+# title must end with one or more spaces followed by the schema version, which must begin with "v"
+title: ATAOS v3
+description: Schema for ATAOS configuration files
+type: object
+properties:
+  correction_frequency:
+    description: The frequency which corrections should be applied (Hz).
+    type: number
+    default: 1
+  m1:
+    description: List of polynomial coefficients for M1 correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  m1_lut_elevation_limits:
+    description: >-
+        Elevation limits for the validity of the M1 pressure LUT. Any values
+        outside this range you be assigned the value at this position.
```

Comment:
...outside this range will be assigned....

Reply:
So this is a little dangerous actually as it can lead to mirror lifting. For example, if you put the lower limit at 20 degrees, then when you go to 10 the pressure will be set for 20, which might cause a lift. Maybe it doesn't matter?
Another way to do this would be to just make the function linear to zero?

Reply:
I think you are right that it might be a bit prone to lifting the mirror, especially at these lower elevations. It could as well switch to a linear fit bellow the limit. Probably not going to zero though, maybe the user provide an inclination and we computer the zero-point from the value in the limit?